google/cloud/spanner_v1/database.py

```diff
@@ -796,9 +801,11 @@ def run_in_transaction(self, func, *args, **kw):
 
         :type kw: dict
         :param kw: (Optional) keyword arguments to be passed to ``func``.
-                   If passed, "timeout_secs" will be removed and used to
+                   If passed,
+                   "timeout_secs" will be removed and used to
                    override the default retry timeout which defines maximum timestamp
                    to continue retrying the transaction.
+                   "max_commit_delay" will be removed and used to set the max_commit_delay for the request.
```

Comment:
Same as above comment. Can we mention the allowed range of max_commit_delay as this is public interface through which customers can pass this option.

Reply:
Done.